### ISSUE TITLE
Reanimate won't compile on MacOS Big Sur

### ISSUE THREAD

**Issue by joseedil** [reanimate/reanimate#199]
Trying 

```
$ git clone https://github.com/reanimate/reanimate.git
$ cd reanimate/
$ stack build
```

on macOS Big Sur version 11.1 won't finish with an error. All the dependencies seem to compile and install properly.

```
reanimate              > configure (lib)
Configuring reanimate-1.1.3.2...
reanimate              > build (lib)
Preprocessing library for reanimate-1.1.3.2..
Building library for reanimate-1.1.3.2..
[ 1 of 58] Compiling Detach
[ 2 of 58] Compiling Geom2D.CubicBezier.Linear
[ 3 of 58] Compiling Paths_reanimate
[ 4 of 58] Compiling Reanimate.ColorMap
[ 5 of 58] Compiling Reanimate.ColorSpace
[ 6 of 58] Compiling Reanimate.Constants
[ 7 of 58] Compiling Reanimate.Driver.Magick
[ 8 of 58] Compiling Reanimate.Driver.Server
[ 9 of 58] Compiling Reanimate.Driver.Daemon
[10 of 58] Compiling Reanimate.Ease
[11 of 58] Compiling Reanimate.ColorComponents
[12 of 58] Compiling Reanimate.Math.Common
[13 of 58] Compiling Reanimate.Math.Triangulate
[14 of 58] Compiling Reanimate.Math.SSSP
[15 of 58] Compiling Reanimate.Math.Polygon
[16 of 58] Compiling Reanimate.Memo
[17 of 58] Compiling Reanimate.Misc
[18 of 58] Compiling Reanimate.Driver.Check
[19 of 58] Compiling Reanimate.Parameters
[20 of 58] Compiling Reanimate.Svg.LineCommand
[21 of 58] Compiling Reanimate.Transform
[22 of 58] Compiling Reanimate.Svg.BoundingBox
[23 of 58] Compiling Reanimate.Svg.Constructors
[24 of 58] Compiling Reanimate.Svg.Unuse
[25 of 58] Compiling Reanimate.Animation
[26 of 58] Compiling Reanimate.Scene.Core
[27 of 58] Compiling Reanimate.Scene.Var
[28 of 58] Compiling Reanimate.Render
[29 of 58] Compiling Reanimate.Driver.CLI
[30 of 58] Compiling Reanimate.Driver
[31 of 58] Compiling Reanimate.Cache
[32 of 58] Compiling Reanimate.Raster
[33 of 58] Compiling Reanimate.External
[34 of 58] Compiling Reanimate.Povray
[35 of 58] Compiling Reanimate.Blender
[36 of 58] Compiling Reanimate.Svg
[37 of 58] Compiling Reanimate.PolyShape
[38 of 58] Compiling Reanimate.Morph.Common
[39 of 58] Compiling Reanimate.Morph.Rotational
[40 of 58] Compiling Reanimate.Morph.Cache
[41 of 58] Compiling Reanimate.Morph.Linear
[42 of 58] Compiling Reanimate.Math.Balloon
[43 of 58] Compiling Reanimate.LaTeX
[44 of 58] Compiling Reanimate.Effect
[45 of 58] Compiling Reanimate.Debug
[46 of 58] Compiling Reanimate.Builtin.TernaryPlot
[47 of 58] Compiling Reanimate.Builtin.Images
[48 of 58] Compiling Reanimate.Builtin.Documentation
[49 of 58] Compiling Reanimate.Builtin.CirclePlot
[50 of 58] Compiling Reanimate.Transition
[51 of 58] Compiling Reanimate.Scene.Sprite
[52 of 58] Compiling Reanimate.Scene.Object
[53 of 58] Compiling Reanimate.Scene
[54 of 58] Compiling Reanimate
[55 of 58] Compiling Reanimate.GeoProjection
[56 of 58] Compiling Reanimate.Builtin.Slide
[57 of 58] Compiling Reanimate.Builtin.Flip
<command line>: can't load framework: Cocoa (not found)
Completed 125 action(s).

--  While building package reanimate-1.1.3.2 (scroll up to its section to see the error) using:
      /Users/jose.edil/.stack/setup-exe-cache/x86_64-osx/Cabal-simple_mPHDZzAJ_3.0.1.0_ghc-8.8.4 --builddir=.stack-work/dist/x86_64-osx/Cabal-3.0.1.0 build lib:reanimate --ghc-options " -fdiagnostics-color=always"
    Process exited with code: ExitFailure 1
```

**Comment by joseedil** [comment]
@Lemmih, seems to be a GHC issue that was fixed on 8.10.3.

https://gitlab.haskell.org/ghc/ghc/-/merge_requests/3689

Could make it compile by using `resolver: lts-17.4`. 
May I suggest making this the default?

**Comment by Lemmih** [comment]
Done.
